clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderProducerCallback.java

```diff
@@ -270,7 +258,7 @@ public void setChunkingInfo(
         record.setRmdManifest(chunkedRmdManifest);
       } else if (partitionConsumptionState.isEndOfPushReceived()) {
         String msg = "Transient record is missing when trying to update value/RMD manifest for topic "
-            + ingestionTask.getKafkaVersionTopic() + " partition: " + subPartition;
+            + ingestionTask.getKafkaVersionTopic() + " partition: " + partition;
```

Comment:
can replace it with `Utils.getReplicaId(ingestionTask.getKafkaVersionTopic(), partition)`